protobuf/BUILD

```diff
@@ -76,6 +76,20 @@ proto_library(
     ]
 )
 
+filegroup(
+    name = "proto-raw-buffers",
+    srcs = [
+        "answer.proto",
+        "concept.proto",
+        "database.proto",
+        "grakn.proto",
+        "options.proto",
+        "query.proto",
+        "session.proto",
+        "transaction.proto",
+    ]
+)
```

Comment:
The libraries we currently have pre-compile the .proto files, which is inconvenient as we wish to recompile them. Where we could (and in future should) build a full blown bazel rule extracting .src_file from the proto_library targets, for the time being providing a filegroup is equally legitimate

Reply:
Totally with you on this one - let's raise a GitHub issue for it (in `graknlabs/protocol`) and possibly put a `TODO` in the source code linking to the issue so it doesn't get lost.